### PR TITLE
RabbitMQ TLS + Delegator

### PR DESCRIPTION
**Pull request by bonedaddy**
Allows for the usage of client-based certificate authentication with rabbitmq
Allows for specifying delegator ports

### CODE REVIEW

File: types.go [hunk 1]
```diff
@@ -186,4 +190,8 @@ type Orchestrator struct {
 		CertPath string `json:"cert"`
```
Comment (bobheadxi) [inline types.go:191]: can you rename the above to `Nexus`?

File: types.go [hunk 2]
```diff
@@ -186,4 +190,8 @@ type Orchestrator struct {
...
 		KeyPath  string `json:"key"`
 	} `json:"tls"`
+	Delegator struct {
+		Host string `json:"host"`
```
Comment (bobheadxi) [inline types.go]: this `Host` will always be the same as the Orchestrator/Nexus host, so a little redundant

**Review by bobheadxi — CHANGES_REQUESTED** [review]

File: types.go [hunk 3]
```diff
@@ -2,20 +2,20 @@ package config
...
+	IPFSCluster `json:"ipfs_cluster,omitempty"`
+	MINIO       `json:"minio,omitempty"`
+	RabbitMQ    `json:"rabbitmq,omitempty"`
+	AWS         `json:"aws,omitempty"`
+	Sendgrid    `json:"sendgrid,omitempty"`
+	Ethereum    `json:"ethereum,omitempty"`
+	Wallets     `json:"wallets,omitempty"`
+	APIKeys     `json:"api_keys,omitempty"`
+	Endpoints   `json:"endpoints,omitempty"`
+	Nexus       `json:"Nexus,omitempty"`
```
Comment (bobheadxi) [inline types.go]: to be in line with the others:

```suggestion
	Nexus       `json:"nexus,omitempty"`
```

File: types.go (hunk 2 above)
Comment (bonedaddy) [reply types.go]: Yup good point

File: types.go (hunk 3 above)
Comment (bonedaddy) [reply types.go]: good catch

**Review by bobheadxi — APPROVED** [review]
thanks!

**Pull request merged**